tools/otci/otci/otci.py

```diff
@@ -2586,10 +2606,51 @@ def diag_radio_sleep(self):
         """Enter radio sleep mode."""
         self.execute_command('diag radio sleep')
 
+    def diag_radio_enable(self):
+        """Enable the radio."""
+        self.execute_command('diag radio enable')
+
+    def diag_radio_disable(self):
+        """Disable the radio."""
+        self.execute_command('diag radio disable')
+
     def diag_radio_receive(self):
         """Set radio to receive mode."""
         self.execute_command('diag radio receive')
 
+    def diag_radio_receive_number(self, number: int):
+        """Set radio to receive mode and receive specified number of packets."""
+
+        output = self.execute_command(f'diag radio receive {number} lpr')
+
+        if len(output) != number:
+            raise UnexpectedCommandOutput(output)
+
+        result = []
+
+        for line in output:
+            data = line.split(',')
+            result.append({
+                'rssi': int(data[1].split(":")[1]),
+                'lqi': int(data[2].split(":")[1]),
+                'len': int(data[3].split(":")[1]),
+                'psdu': data[4].split(":")[1],
+            })
```

Comment:
Better to document some example data so it will be easier to understand how it's parsed.